server-sample.yaml

```
@@ -44,6 +44,9 @@ livekit:
     # Kubernetes Secret containing TLS cert for <turn.myhost.com>
     # See https://docs.livekit.io/deploy/kubernetes/#importing-ssl-certificates
     secretName: <tlssecret>
+    # set the Kubernetes serviceType for the TURN service. By default it sets it to "LoadBalancer"
+    # See kubernetes serviceTypes on official documentation: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
+    service_type: "LoadBalancer"
```

Just a nit, since this isn't a livekit config value (like the other values above), let's keep it in kube naming convention.

```suggestion
    serviceType: "LoadBalancer"
```